File: ios/Objective-C/HelloWorld/HelloWorld/DDNDataManager.m
//
//  HelloWorld.m
//  HelloWorld
//
//  Created by admin on 2022/3/4.
//  Copyright © 2022 dynamsoft. All rights reserved.
//

#import "DDNDataManager.h"

@implementation DDNDataManager

+ (DDNDataManager *)instance{
    static DDNDataManager *instance = nil;
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        instance = [super allocWithZone:NULL];
    });
    return instance;
}

@end

File: ios/Swift/HelloWorld/HelloWorld/DDNDataManager.swift
//
//  DDNDataManager.swift
//  HelloWorld
//
//  Copyright © Dynamsoft. All rights reserved.
//

import Foundation
import DynamsoftDocumentNormalizer

class DDNDataManager: NSObject {
    var ddn: DynamsoftDocumentNormalizer!
    var resultImage: UIImage!
    var quadArr: [iDetectedQuadResult]!
    var imageData: iImageData!

    static let instance = DDNDataManager()
}
